devtools/common/src/main/filtered/extensions.json

```diff
@@ -432,6 +437,7 @@
   },
   {
     "name": "Swagger UI",
+    "shortName": "swagger",
```

Comment:
Do we really want the Swagger UI component to trump all other "swagger" stuff? Isn't swagger more than just a UI component?

Reply:
I don't really know. We always refer to _swagger_. Might be better to not use it there.

Reply:
The other bit that is `swagger` from the community we have in the form of `openapi`, so I don't think we'd add a pure `swagger` extension. Though it may be confusing to some to not have UI in there

Reply:
Yes, I've removed the short name.